### DISCUSSION TITLE
How likely to have Apple Vision OS support on dotnet MAUI?

### DISCUSSION THREAD

**Discussion by bonniemathew** [dotnet/maui#16883] [General]
Now that Apple has introduced the Apple Vision PRO, can we expect dotnet MAUI to support Vision OS platform?

I know its too early to ask, but just wanted to know whether there is plan for it in the roadmap.

**Comment by RobTF** [comment]
Don't give them any ideas! Let's get iOS and Android working first ;)

It's already a huge distraction with them trying to support stuff that a lot of developers don't care much about as other alternatives on .NET for those platforms exist (such as Windows and Mac).
